### PR TITLE
[Qwen3 VL] reduce memory use of debugmodels

### PR DESCRIPTION
**Pull request by shuhuayu**
To fix ci ooms on a10g machines: https://github.com/pytorch/torchtitan/issues/3025.

### CODE REVIEW

File: torchtitan/models/qwen3_vl/config_registry.py [hunk 1]
```diff
@@ -157,10 +157,10 @@ def qwen3_vl_30b_a3b() -> Trainer.Config:
     return Trainer.Config(
         hf_assets_path="./assets/hf/Qwen3-VL-30B-A3B-Instruct",
         tokenizer=MultiModalTokenizer.Config(**QWEN3_VL_SPECIAL_TOKENS),
-        model_spec=model_registry("30B-A3B", moe_comm_backend="standard"),
+        model_spec=model_registry("30B-A3B"),
         dataloader=_qwen3_vl_dataloader("cc12m"),
         optimizer=OptimizersContainer.Config(lr=3e-4),
-        lr_scheduler=LRSchedulersContainer.Config(warmup_steps=600),
+        lr_scheduler=LRSchedulersContainer.Config(warmup_steps=20),
```
Comment (tianyu-l) [inline torchtitan/models/qwen3_vl/config_registry.py:163]: why changing this?

**Review by tianyu-l — APPROVED** [review]
one comment

File: torchtitan/models/qwen3_vl/config_registry.py (hunk 1 above)
Comment (shuhuayu) [reply torchtitan/models/qwen3_vl/config_registry.py:163]: This is to fix a previous bug on `warmup_steps=600` but total `steps=100`. For all real models, unified to the same test run config `warmup_steps=20`, total `steps = 1000`. For pre-train studies, the warmup steps typically is `1%` to `2%` of the total steps, e.g., https://arxiv.org/abs/2305.14342.
